src/Components/NotesPage/FilterBar/FilterBar.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { useFilter } from '../../../context';
+import { priorities } from '../../../data/priorities';
+import { tags } from '../../../data/tags';
+import './FilterBar.css';
+const FilterDropDownbutton = (props) => {
+	return (
+		<div className='dropdown'>
+			<button className='btn btn-filter btn-primary dropdown-btn'>{props.buttonName}</button>
+			<div style={{ transform: `translateX(-${props.scroll}px)` }} className='dropdown-content'>
+				{props.children}
+			</div>
+		</div>
+	);
+};
+
+const FilterBar = () => {
+	const { FilterState, FilterDispatch } = useFilter();
```

Comment:
Suggestion: Dont make the state and dispatch function capital as they are not components.

Reply:
Done